test/regression.py

```diff
@@ -165,6 +165,16 @@ def testADF_ADF2016_fa2_adf_out(logfile):
 
 # DALTON #
 
+def testDALTON_DALTON_2013_dvb_td_normalprint_out(logfile):
+    """This original unit test prints a DFT-specific version of the excitation

```

Comment:
Could you reiterate why we do not parse the DFT-specific version?

Reply:
The field that was parsed (https://github.com/cclib/cclib/pull/680/files#diff-88f10a78215b8cf004128ef198e5762bL1197) is incorrect, and the correct field (the scaled one) that matches what other programs print isn't available in the DFT-specific output.

Reply:
Could you make this clear in the docstring here, and/or in the parser code? I think these types of things are worth documenting, so we don't trip over them again in the future.

Reply:
Done in https://github.com/cclib/cclib/pull/680/commits/bfd9a8d8bbaa1b34e2af4d57e30174fd7e34d964

Reply:
Thanks!